scriptmodules/emulators/simcoupe.sh

```diff
@@ -13,20 +13,25 @@ rp_module_id="simcoupe"
 rp_module_desc="SimCoupe SAM Coupe emulator"
 rp_module_help="ROM Extensions: .dsk .mgt .sbt .sad\n\nCopy your SAM Coupe games to $romdir/samcoupe."
 rp_module_licence="GPL2 https://raw.githubusercontent.com/simonowen/simcoupe/master/License.txt"
+rp_module_repo="git https://github.com/simonowen/simcoupe.git master :_get_branch_simcoupe"
```

Comment:
I think there's an extra `master` here.

Reply:
The additional master was wrong. But initializing `$branch` to master in the function seems correct unless that's not the default branch?  The function hook is only needed for stretch support so will remove in the future I guess but let me know if I've missed something. For now have removed master from md_module_repo variable.